### PR TITLE
doc: add an expanding branch navigator

### PR DESCRIPTION
**Pull request by jorgeorpinel**
- [ ] Should prob. be applied to every branch.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,40 @@
...
+  variation features the
+  [make_checkpoint](https://dvc.org/doc/api-reference/make_checkpoint) function
+  from DVC's Python API.
+- Contrastingly, the
+  [signal file](https://github.com/iterative/dvc-checkpoints-mnist/tree/signal_file)
+  scenario shows how to make your own signal files (applicable to any
+  programming language).
+- Finally, our
+  [full pipeline](https://github.com/iterative/dvc-checkpoints-mnist/tree/full_pipeline)
+  scenario elaborates on the full-featured usage with a more advanced process.
```
Comment (jorgeorpinel) [inline README.md:39]: I kind of made this one up 😋

File: README.md [hunk 2]
```diff
@@ -1,11 +1,40 @@
...
-This example DVC project uses [dvclive](https://dvc.org/doc/dvclive) to generate
+This example DVC project uses [DVCLive](https://dvc.org/doc/dvclive) to generate
 [checkpoints](https://dvc.org/doc/api-reference/make_checkpoint) for iterative
 model training. The model is a simple convolutional neural network (CNN)
 classifier trained on the [MNIST](http://yann.lecun.com/exdb/mnist/) data of
 handwritten digits to predict the digit (0-9) in each image.
 
+<details>
+
+<summary>🔄 Switch between checkpoint scenarios</summary>
```
Comment (jorgeorpinel) [inline README.md]:
```suggestion
This example DVC project demonstrates the different ways to employ
[Checkpoint Experiments](https://dvc.org/doc/user-guide/experiment-management#checkpoints-in-source-code) with DVC.

This scenario uses [DVCLive](https://dvc.org/doc/dvclive) to generate
[checkpoints](https://dvc.org/doc/api-reference/make_checkpoint) for iterative
model training. The model is a simple convolutional neural network (CNN)
classifier trained on the [MNIST](http://yann.lecun.com/exdb/mnist/) data of
handwritten digits to predict the digit (0-9) in each image.

<details>

<summary>🔄 Switch between scenarios</summary>
```

File: README.md [hunk 3]
```diff
@@ -1,11 +1,43 @@
...
 
-This example DVC project uses [dvclive](https://dvc.org/doc/dvclive) to generate
+This example DVC project demonstrates the different ways to employ
+[Checkpoint Experiments](https://dvc.org/doc/user-guide/experiment-management#checkpoints-in-source-code) with DVC.
+
+This scenario uses [DVCLive](https://dvc.org/doc/dvclive) to generate
 [checkpoints](https://dvc.org/doc/api-reference/make_checkpoint) for iterative
 model training. The model is a simple convolutional neural network (CNN)
 classifier trained on the [MNIST](http://yann.lecun.com/exdb/mnist/) data of
 handwritten digits to predict the digit (0-9) in each image.
```
Comment (jorgeorpinel) [inline README.md]:
```suggestion
This example DVC project demonstrates the different ways to employ
[Checkpoint Experiments](https://dvc.org/doc/user-guide/experiment-management#checkpoints-in-source-code)
with DVC.

This scenario uses [DVCLive](https://dvc.org/doc/dvclive) to generate
[checkpoints](https://dvc.org/doc/api-reference/make_checkpoint) for iterative
model training. The model is a simple convolutional neural network (CNN)
classifier trained on the [MNIST](http://yann.lecun.com/exdb/mnist/) data of
handwritten digits to predict the digit (0-9) in each image.
```

File: README.md [hunk 4]
```diff
@@ -1,11 +1,43 @@
...
+  variation features the
+  [make_checkpoint](https://dvc.org/doc/api-reference/make_checkpoint) function
+  from DVC's Python API.
+- Contrastingly, the
+  [signal file](https://github.com/iterative/dvc-checkpoints-mnist/tree/signal_file)
+  scenario shows how to make your own signal files (applicable to any
+  programming language).
+- Finally, our
+  [full pipeline](https://github.com/iterative/dvc-checkpoints-mnist/tree/full_pipeline)
+  scenario elaborates on the full-featured usage with a more advanced process.
```
Comment (jorgeorpinel) [inline README.md]:
```suggestion
This repo has several
[branches](https://github.com/iterative/dvc-checkpoints-mnist/branches) to this
that show different methods for using checkpoints on a similar pipeline:

- The [live](https://github.com/iterative/dvc-checkpoints-mnist/edit/live)
  scenario introduces full-featured checkpoint usage — integrating with
  [DVCLive](https://github.com/iterative/dvclive).
- The [basic](https://github.com/iterative/dvc-checkpoints-mnist/tree/basic)
  scenario uses single-checkpoint experiments to illustrate how checkpoints work
  in a simple way.
- The
  [Python-only](https://github.com/iterative/dvc-checkpoints-mnist/tree/make_checkpoint)
  variation features the
  [make_checkpoint](https://dvc.org/doc/api-reference/make_checkpoint) function
  from DVC's Python API.
- Contrastingly, the
  [signal file](https://github.com/iterative/dvc-checkpoints-mnist/tree/signal_file)
  scenario shows how to make your own signal files (applicable to any
  programming language).
- Finally, our
  [full pipeline](https://github.com/iterative/dvc-checkpoints-mnist/tree/full_pipeline)
  scenario elaborates on the full-featured usage with a more advanced process.
```

**Comment by dberenbaum** [comment]
Thanks, @jorgeorpinel! I've been a bit hesitant to do this in part because I don't think `live` is that obvious of a default branch. Can we first decide on the branch structure and the default branch to put this in, and then we can add this?

**Comment by jorgeorpinel** [comment]
Not sure either about the default branch, but I'd put this in every branch's REDAME anyway.

File: README.md (hunk 1 above)
Comment (dberenbaum) [reply README.md:39]: Yeah, I'm leaning towards deleting this branch. Thoughts?

Comment (jorgeorpinel) [reply README.md:39]: I didn't have a chance to try this one so I can't give you an answer. From a high level it seem like an extra, up to you

**Pull request merged**